addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -169,14 +169,18 @@ def _l10n_it_post_invoices_step_1(self, invoices):
             try:
                 responses = self._l10n_it_edi_upload([i['data'] for i in to_send.values()], proxy_user)
             except AccountEdiProxyError as e:
-                return {invoice: {'error': e.message, 'blocking_level': 'error'} for invoice in invoices}
+                return {invoice: {
+                    'attachment' : invoice.l10n_it_edi_attachment_id,
+                    'error': e.message,
+                    'blocking_level': 'error'} for invoice in invoices}
```

Comment:
It feels really wrong to have an attachment when there is an error :thinking: 

____________
On another topic, for the style:

```suggestion
                return {
                    invoice: {
                        'attachment' : invoice.l10n_it_edi_attachment_id,
                        'error': e.message,
                        'blocking_level': 'error'
                    } for invoice in invoices
                }
```
or
```suggestion
                return {
                    invoice: {
                        'attachment' : invoice.l10n_it_edi_attachment_id,
                        'error': e.message,
                        'blocking_level': 'error'
                    } 
                    for invoice in invoices
                }
```
or (but this is my least favorite)
```suggestion
                return {invoice: {
                    'attachment' : invoice.l10n_it_edi_attachment_id,
                    'error': e.message,
                    'blocking_level': 'error'
                } for invoice in invoices}
```

Reply:
The user may want to send the invoice by other means if it's really urgent